internal/services/storage/migration/account.go

```diff
@@ -204,3 +211,810 @@ func accountSchemaForV0AndV1() map[string]*pluginsdk.Schema {
+		},
+
+		"primary_blob_host": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"secondary_blob_endpoint": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"secondary_blob_host": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"primary_queue_endpoint": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"primary_queue_host": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"secondary_queue_endpoint": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"secondary_queue_host": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"primary_table_endpoint": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"primary_table_host": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"secondary_table_endpoint": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"secondary_table_host": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"primary_web_endpoint": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"primary_web_host": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"secondary_web_endpoint": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"secondary_web_host": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"primary_dfs_endpoint": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"primary_dfs_host": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"secondary_dfs_endpoint": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"secondary_dfs_host": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"primary_file_endpoint": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"primary_file_host": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"secondary_file_endpoint": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"secondary_file_host": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"primary_access_key": {
+			Type:      pluginsdk.TypeString,
+			Sensitive: true,
+			Computed:  true,
+		},
+
+		"secondary_access_key": {
+			Type:      pluginsdk.TypeString,
+			Computed:  true,
+			Sensitive: true,
+		},
+
+		"primary_connection_string": {
+			Type:      pluginsdk.TypeString,
+			Computed:  true,
+			Sensitive: true,
+		},
+
+		"secondary_connection_string": {
+			Type:      pluginsdk.TypeString,
+			Computed:  true,
+			Sensitive: true,
+		},
+
+		"primary_blob_connection_string": {
+			Type:      pluginsdk.TypeString,
+			Computed:  true,
+			Sensitive: true,
+		},
+
+		"secondary_blob_connection_string": {
+			Type:      pluginsdk.TypeString,
+			Computed:  true,
+			Sensitive: true,
+		},
+
+		"tags": {
+			Type:         pluginsdk.TypeMap,
+			Optional:     true,
+			ValidateFunc: validate.StorageAccountTags,
```

Comment:
fwiw you don't need these for a state migration - only the Type and Required/Optional/Computed iirc

```suggestion
```